pkgs/tools/inputmethods/ibus/default.nix

```diff
@@ -29,12 +30,14 @@ stdenv.mkDerivation rec {
     "--disable-memconf"
     "--enable-ui"
     "--enable-python-library"
+    "--with-emoji-json-file=${nodePackages.emojione}/lib/node_modules/emojione/emoji.json"
```

Comment:
Just `emojione` and it actually use only one file from it.
Neither `emojione` or `nodejs` are in the `nix-store -qR` list.